apps/block_scout_web/lib/block_scout_web/templates/block/_tile.html.eex

```diff
@@ -20,7 +20,7 @@
             #<%= @block %>
           <% end %>
       <% end %>
-      <span class="tile-status-label font-weight-400" data-test="transaction_type"><%= @block_type %></span>
+      <span class="tile-status-label font-weight-400" data-test="transaction_type">Batch</span>
```

Comment:
> It looks like this was a i18n variable before right?

What makes you think so? I think they just forgot calling `gettext` here.

> Maybe it's better to add it as a i18n than hardcoding it here?

I went this way because the file `apps/block_scout_web/priv/gettext/en/LC_MESSAGES/default.po` is automatically generated from the template `apps/block_scout_web/priv/gettext/default.pot`. Manually adding new entries didn't seem like a good idea to me.

Also, like this, we avoid changing 2 files.

Reply:
Ahá I understand! Then it's perfect :)